tests/ttsim/interface_dag_elements/test_failures.py

```diff
@@ -842,6 +844,7 @@ def test_fail_if_p_id_does_not_exist_via_main(backend):
             input_data={"tree": data},
             policy_environment={},
             tt_targets={"tree": {}},
+            date=datetime.date(2025, 1, 1),
```

Comment:
Looks like the variants of `test_fail_if_p_id_does_not_exist` and `test_fail_if_p_id_is_missing` are exact duplicates?